test/pytorch_test_base.py

```diff
@@ -180,10 +180,7 @@
         'test_index_mem_overlap',  # doesn't raise
         'test_maximum_minimum_complex',  # doesn't raise
         'test_maximum_minimum_float_xla_bfloat16',  # precision
-        'test_maximum_minimum_type_promotion_xla_bfloat16*',  # doesn't raise
-        'test_maximum_minimum_type_promotion_xla_float16*',  # doesn't raise
-        'test_maximum_minimum_type_promotion_xla_*_bfloat16',  # doesn't raise
-        'test_maximum_minimum_type_promotion_xla_*_float16',  # doesn't raise
+        'test_maximum_minimum_type_promotion_xla_*bfloat16*',  # doesn't raise
```

Comment:
Hmm, this is not a valid regex. Should use `.*` instead of `*` ...

Reply:
yup.. you right, I confuse the wildcard with the regex 🤣